index.js

```diff
@@ -73,7 +74,8 @@ function choo (opts) {
   function render (state, data, prev, name, createSend) {
     if (!_frame) {
       _frame = nanoraf(function (state, prev) {
-        const newTree = _router(state.location.pathname, state, prev)
```

Comment:
this would indeed also mean that we first check for `#something` on the page, and kinda cross our fingers duplicates don't exist in the router that would cause a route change. When using hashes for only one type of navigation this conflict should never occur, and if for some reason both types of routing need to exist the addition of the `state.location.query` parameter could provide an escape hatch for the routing (as GH does on their profile page). Reckon this is good enough? - if we go forward with this it would probably be worth documenting in detail haha